src/lakefs_spec/commithook.py

```diff
@@ -35,7 +35,7 @@ def Default(fs_event: FSEvent, ctx: HookContext) -> CommitCreation:
     elif fs_event == FSEvent.RM:
         action = "Remove"
     else:
-        raise ValueError(f"unexpected file system event {fs_event!r}")
+        raise ValueError(f"unexpected file system event {str(fs_event)!r}")
```

Comment:
Maybe just drop the `!r` flag? `f"... {fs_event}"` should work fine.

Reply:
That's needed for the quotes around the event name - which I want to make clear as the input to the hook in the error:

`ValueError: Unexpected file system event 'foobar'` vs. `ValueError: Unexpected file system event foobar`

I'm trying to do the former. What do you think?

Reply:
Phew. I'd just go with f'..."{fs_event}"', which makes it explicit that you're after the quotes.

Reply:
Done in a5206aa.